src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -57,4 +58,21 @@ export default interface VideoUplinkBandwidthPolicy {
    * Returns the selected [[MediaTrackConstraints]] to update
    */
   chooseMediaTrackConstraints(): MediaTrackConstraints;
+
+  /**
+   * Set a reference to the current transceiver controller.
+   * This will be called during connection and if unified plan is enabled.
+   * This method should not throw.
+   * @param {TransceiverController} transceiverController - The transceiver controller
+   */
+  setTransceiverController?(transceiverController: TransceiverController): void;
```

Comment:
How do I know when to end my subscriptions? Does my policy need to subscribe to `AudioVideoObserver`?

Reply:
Sorry what do you mean by subscription? This method is just to be set by audio video controller after connection right?

Reply:
I mean: say my policy subscribes to the active speaker detector, or some other observer interface.

It needs to unsubscribe from that interface when the call ends, and additionally drop its reference to the transceiver controller, in order to prevent leaks of call objects.

How does it know?

Reply:
The policy should not need to subscribe to active speaker detector audio video observer. The audio video controller decides to set whenever the local video has bandwidth priority and call `setHasBandwidthPriority`.

Reply:
I mean: I want to implement a downlink policy that makes sure that the active talker has the highest priority. How do I know who's the active talker? I make my policy subscribe to that.

But regardless, the point stands for other kinds of linkages and the leakage of the transceiver controller itself.